### PR TITLE
Chore/cleanup

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: .phpunit.result.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...\\Policy\\OrmResolverTest::testGetPolicyUnknownObject":0.004,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyUnknownEntity":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedEntity":0,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedPluginEntityAppOveride":0,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedPluginEntityPluginOveride":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedPluginEntity":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyDefinedTable":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyQueryForDefinedTable":0.001,"Authorization\\Test\\TestCase\\Policy\\OrmResolverTest::testGetPolicyUnknownTable":0.002,"Authorization\\Test\\TestCase\\Policy\\ResolverCollectionTest::testEmptyCollection":0,"Authorization\\Test\\TestCase\\Policy\\ResolverCollectionTest::testMissingPolicy":0.001,"Authorization\\Test\\TestCase\\Policy\\ResolverCollectionTest::testGetPolicy":0,"Authorization\\Test\\TestCase\\Middleware\\RequestAuthorizationMiddlewareTest::testUnauthorizedHandlerSuppress":0}}
```
Comment (dereuromark) [inline .phpunit.result.cache:1]: Was this file pushed on purpose?

Comment (ADmad) [reply .phpunit.result.cache:1]: Doh.. no, no idea how that got in, will clean it up.

Comment (dereuromark) [reply .phpunit.result.cache:1]: .gitattributes doesnt contain the `.phpunit.result.cache` file yet.

Comment (ADmad) [reply .phpunit.result.cache:1]: True, but I almost always use `git commit -a` to only commit modifications to existing files only.

**Pull request merged**